src/components/TagInput/TagInput.styles.ts

```diff
@@ -0,0 +1,104 @@
+import { BaseProps } from '@/types';
+
+type InputConteinerConfig = BaseProps & {
+  isValidationError: BaseProps;
+};
+
+export type TagInputConfig = BaseProps & {
+  innerElements?: {
+    input?: BaseProps;
+    inputContainer?: InputConteinerConfig;
+    label?: BaseProps;
+    helperText?: BaseProps;
+  };
+};
+
+export const defaultConfig = {
+  display: 'flex',
+  flexDirection: 'column',
+  height: '40px',
+  gap: '$space-component-gap-small',
+  w: '320px',
+  padding:
+    '$space-component-padding-small $space-component-padding-large $space-component-padding-small $space-component-padding-small',
+  flexShrink: 0,
+  innerElements: {
+    inputContainer: {
+      w: '100%',
+      h: '$size-medium',
+      borderRadius: '$border-radius-large',
+      border: '1px solid $color-interaction-border-neutral-normal',
+      display: 'flex',
+      gap: '$space-component-gap-small',
+      alignItems: 'center',
+      bg: '$color-interaction-background-formField',
+      borderColor: 'rgb(207, 214, 221)',
+      p: '$space-component-padding-small $space-component-padding-large $space-component-padding-small $space-component-padding-small',
+      flexShrink: '0',
+      ringInset: 'true',
+      ring: '$border-width-small',
+      boxShadow: '0 0 #0000',
```

Comment:
I don't know where that comes from in Figma. we usually have box shadows tokens available but I can't find any usage of box-shadow here

Reply:
it was unnecessary - I removed it